AppDB/cassandra_env/cassandra_interface.py

```diff
@@ -165,16 +171,23 @@ def batch_delete(self, table_name, row_keys, column_names=[]):
     if not isinstance(table_name, str): raise TypeError("Expected a str")
     if not isinstance(row_keys, list): raise TypeError("Expected a list")
 
-    path = ColumnPath(table_name)
+    row_keys_bytes = [bytearray(row_key) for row_key in row_keys]
+
+    statement = 'DELETE FROM "{table}" WHERE {key} IN %s'.\
+      format(
+        table=table_name,
+        key=ThriftColumn.KEY
+      )
+    query = SimpleStatement(statement,
+                            consistency_level=ConsistencyLevel.QUORUM)
```

Comment:
Yes. I'm not sure the best way to do this. To me, it would make the most sense to pass it in as a parameter to the function because the most likely scenario would be that someone would want to set it to ONE for most index queries, but not range scans involving ancestor queries.